packages/postcss-themes/src/index.js

```diff
@@ -0,0 +1,55 @@
+const postcss = require( 'postcss' );
```

Comment:
it doesn't need to be transpiled, should we put in the top folder?

Reply:
I personally prefer to be consistent. Maybe we should introduce an exclusion rule in our potential build script instead

Reply:
I think we should raise this in the #core-js chat, I'm leaning towards consistency and thinking we should put all `@wordpress` packages into `/src` folders.

With more and more packages being released and the increase in contributors looking to _grok_ all of @wordpress' packages, consistency in package layout and formatting I think would help new users and contributors when first exploring all the new shiny things WordPress are putting forward.

''Not a blocker, so can be decided at a later date''

Reply:
> ''Not a blocker, so can be decided at a later date''

Agreed 👍